listenbrainz_spark/utils.py

```diff
@@ -42,15 +43,20 @@ def read_files_from_HDFS(path):
         df = listenbrainz_spark.sql_context.read.parquet(path)
         return df
     except AnalysisException as err:
-        raise AnalysisException('Cannot read "{}" from HDFS: {}\n'.format(path, type(err).__name__),
+      raise AnalysisException('Cannot read "{}" from HDFS: {}\n'.format(path, type(err).__name__),
             stackTrace=traceback.format_exc())
     except Py4JJavaError as err:
         raise Py4JJavaError('An error occurred while fetching "{}": {}\n'.format(path, type(err).__name__),
             err.java_exception)
 
-def get_listens():
+def get_listens(d1, d2):
```

Comment:
Function declarations should really have better variable names. Also the args and the docstring do not agree (m1 vs d1)

Reply:
Also, from this function it is unclear if the m2 month will be included. The < says it won't, but this isn't clear from the documentation.